rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -284,38 +293,45 @@ def liquity_staking_balances(
             assets=(A_LQTY, A_LUSD, A_ETH),
         )
 
+    @staticmethod
```

Comment:
Why there was no issue before if it was defined as an instance method and there was no `self` in the list of arguments?